### ISSUE TITLE
Show error message for IOException in frontend of media library

### ISSUE THREAD

**Issue by stollr** [sulu/sulu#6594]
# Problem description

This issue is similar to #5540. When an IOException occurs in the backend, because the OS user running PHP has no write access to the directory `var/uploads` the upload fails, but the admin user does not get any feedback, which is quite confusing.

The HTTP request to `/admin/api/media?collection={id}` for the upload fails with a 500 error and the content
```
{"code":0,"message":"Failed to create \"\/var\/www\/myproject\/var\/uploads\/media\/02\": mkdir(): Permission denied","errors":["Symfony\\Component\\Filesystem\\Exception\\IOException: Failed to create \"\/var\/www\/myproject\/var\/uploads\/media\/02\": mkdir(): Permission denied in \/var\/www\/myproject\/vendor\/symfony\/filesystem\/Filesystem.php:97
Stack trace:
#0 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Media\/Storage\/LocalStorage.php(183): Symfony\\Component\\Filesystem\\Filesystem->mkdir()
#1 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Media\/Storage\/LocalStorage.php(60): Sulu\\Bundle\\MediaBundle\\Media\\Storage\\LocalStorage->createDirectories()
#2 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Media\/Manager\/MediaManager.php(516): Sulu\\Bundle\\MediaBundle\\Media\\Storage\\LocalStorage->save()
#3 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Media\/Manager\/MediaManager.php(322): Sulu\\Bundle\\MediaBundle\\Media\\Manager\\MediaManager->buildData()
#4 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Controller\/MediaController.php(533): Sulu\\Bundle\\MediaBundle\\Media\\Manager\\MediaManager->save()
#5 \/var\/www\/myproject\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/MediaBundle\/Controller\/MediaController.php(414): Sulu\\Bundle\\MediaBundle\\Controller\\MediaController->saveEntity()
#6 \/var\/www\/myproject\/vendor\/symfony\/http-kernel\/HttpKernel.php(152): Sulu\\Bundle\\MediaBundle\\Controller\\MediaController->postAction()
#7 \/var\/www\/myproject\/vendor\/symfony\/http-kernel\/HttpKernel.php(74): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw()
#8 \/var\/www\/myproject\/vendor\/symfony\/http-kernel\/Kernel.php(202): Symfony\\Component\\HttpKernel\\HttpKernel->handle()
#9 \/var\/www\/myproject\/public\/index.php(66): Symfony\\Component\\HttpKernel\\Kernel->handle()
#10 {main}"]}
```

# Proposed solution

Showing an informative error to the admin user.

**Comment by alexander-schranz** [comment]
Can you check if @scarcloud PR #6591 does fix this?

**Comment by niklasnatter** [comment]
i think this should be fixed in #6591. Please feel free to reopen this issue if I am mistaken!
